kuma/static/js/wiki.js

```diff
@@ -896,4 +896,35 @@
         }
     }, false);
 
+    // Insert notice about permanent language switch
+    var changedLocaleTo = localStorage.getItem('changed-locale-to');
+    if (changedLocaleTo) {
+        var locale = JSON.parse(changedLocaleTo),
+            text = gettext('You are now viewing this article in %(locale_name)s.' +
+                           ' Do you always want to view MDN articles in %(locale_name)s?'),
+            html = interpolate('<div id="locale-permanent-banner" class="primary overheadIndicator"><p>' + text +
+                               '<br><button id="locale-permanent-yes" data-locale="%(locale_code)s">' +
+                               gettext('Yes') + '</button> <button id="locale-permanent-no">' +
+                               gettext('No') + '</button></p></div>',
+                {locale_code: locale.code, locale_name: locale.name}, true);
+
+        $('#wiki-content').prepend(html);
+
+        // Add event listener to the buttons
+        $('#locale-permanent-yes').on('click', function () {
+                $.post('./i18n/setlang/', {language: this.dataset.locale})
```

Comment:
``./i18n/setlang/`` should be ``/i18n/setlang/``. Or maybe you mean to serve it at ``/{locale}/i18n/setlang/``?